app/components/Header/index.tsx

```diff
@@ -363,11 +375,9 @@ class Header extends Component<Props, State> {
           >
             <Search
               loggedIn={loggedIn}
-              isOpen={this.props.searchOpen}
               onCloseSearch={this.props.toggleSearch}
               updateUserTheme={this.props.updateUserTheme}
               username={this.props.currentUser?.username}
-              className={utilStyles.hiddenOnMobile}
```

Comment:
Are you sure about this?

Reply:
But you tested it, right?

Reply:
I just checked with prod, and there's no difference. I have no idea why this was here either, since you would want to be able to search on mobile :shrug: